SparkPost/2web2ui
# SA-1606 - Subscription limit on reports

### What Changed
 -Tooltip shows if limit for reports reached
 - Button for creating new report disabled if limit for reports reached

### How To Test
 - Go to report builder
 - Check appteam
 - Create new report button should be working
 - Create new account
 - Enable user ui option `allow_saved_reports`
 - Button should be disabled
 - Tooltip should exist

### To Do
- [x] Check usage (since list isn't guaranteed to be the full list)

### Notes
 - Rebased off of SA-1588. Will mark appropriate file changes
